app/controllers/turbo/notifications_controller.rb

```diff
@@ -6,12 +6,12 @@ class NotificationsController < ApplicationController
       end
```

Comment:
Change the name of the controller maybe to cable, so the path can be quizzes/cable or something like that